### PR TITLE
Example2: TS/Java Gradle multi-module project

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: example2/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.gradle
+build
+ts-frontend/node_modules
```
Comment (vilchik-elena) [inline example2/.gitignore]: @saberduck Do we need this exclusion? We already have `node_modules` in top level `.gitignore`

File: example2/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Sample multi-module project using gradle and TypeScript
```
Comment (vilchik-elena) [inline example2/README.md]: @saberduck `gradle`->`Gradle`? (everywhere:))

File: example2/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Sample multi-module project using gradle and TypeScript 
+
+This project contains two modules, backend written in Java and
```
Comment (vilchik-elena) [inline example2/README.md]: @saberduck why new lines?

File: example2/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Sample multi-module project using gradle and TypeScript 
+
+This project contains two modules, backend written in Java and 
+frontend written in TypeScript. It uses gradle multi-module build, where
+frontend module is built using [gradle-node-plugin](https://github.com/srs/gradle-node-plugin)
```
Comment (vilchik-elena) [inline example2/README.md]: @saberduck Dot at the end is missing

File: example2/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Frontend module handles its dependencies with yarn and is compiled with TypeScript compiler. During SonarQube analysis started by `gradle sonarqube`
```
Comment (vilchik-elena) [inline example2/README.md]: @saberduck it's not clear how to run analysis

File: example2/ts-frontend/src/greeter.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+interface Person {
```
Comment (vilchik-elena) [inline example2/ts-frontend/src/greeter.ts:1]: @saberduck do you have any issue in this code?

Comment (saberduck) [reply example2/ts-frontend/src/greeter.ts:1]: Added violation of https://rules.sonarsource.com/typescript/RSPEC-3786

**Pull request merged**